src/side-navigation/types.js

```diff
@@ -40,12 +40,14 @@ type renderItemT = (
   },
 ) => React.Node;
 
+type mapItemT = (item: Item) => Item;
```

Comment:
please in-line on L68 if it's not used anywhere else

Reply:
Done :)